slashing-protection/src/main/resources/migrations/postgresql/schema

```diff
@@ -0,0 +1,177 @@
+--
```

Comment:
Can we remove this. As it is I don't think it adds value. It contains a lot unnecessary changes such as the flyway database and ownership permissions.

Reply:
gone